jib-core/src/main/java/com/google/cloud/tools/jib/builder/BuildImageSteps.java

```diff
@@ -183,17 +182,18 @@ public void run()
           // Pushes the new image manifest.
           ListenableFuture<Void> pushImageFuture =
               Futures.whenAllSucceed(
-                      pushBaseImageLayerFuturesFuture, pushContainerConfigurationFutureFuture)
+                      pushBaseImageLayerFuturesFuture,
+                      pushContainerConfigurationFutureFuture,
+                      buildContainerConfigurationFutureFuture)
```

Comment:
The dependency on `buildContainerConfigurationFutureFuture` shouldn't be necessary since `pushContainerConfigurationFutureFuture` already depends on it.